cql3/expr/expression.cc

```diff
@@ -322,7 +322,7 @@ bool contains(const column_value& col, const raw_value_view& value, const column
     }
     const auto collection = get_value(col, bag);
     if (collection) {
-        return contains(col.col->type->deserialize(*collection), value);
+        return contains(col.col->type->deserialize(managed_bytes_view(*collection)), value);
```

Comment:
Isn't `collection` already a `managed_bytes_view`?

Reply:
No, `*collection` is a `managed_bytes`. `deserialize` is a template (which takes various views), so we need to make an explicit conversion to a view here.

Reply:
It's sad to force this on users of abstract_type::desrialize, it should have a helper accepting a managed_bytes.